3p/integration.js

```diff
@@ -112,12 +114,8 @@ window.draw3p = function(opt_configCallback) {
   window.context.isMaster = window.context.master == window;
   window.context.data = data;
   window.context.noContentAvailable = triggerNoContentAvailable;
-  if (data.type == 'twitter') {
```

Comment:
Lets keep this, add facebook and update the comment to be just 

`Only make this available to selected embeds until the generic solution is available.`.

The bug implemented the hard work, but we have not fully wired it.